setup.py

```diff
@@ -0,0 +1,25 @@
+import setuptools
```

Comment:
Could you please update the import as follow (if ok): `from setuptools import setup, find_packages`?
Seen in [substra repo](https://github.com/SubstraFoundation/substra/blob/2c43b5c30d78a23e330b7741c9a1a3737818f6f8/setup.py#L19)